signer/migrations/0003__create_tables.sql

```diff
@@ -12,14 +12,14 @@ CREATE TABLE sbtc_signer.bitcoin_blocks (
     block_height BIGINT NOT NULL,
     parent_hash BYTEA NOT NULL,
     confirms BYTEA,
-    created_at TIMESTAMP NOT NULL
+    created_at TIMESTAMPTZ NOT NULL
```

Comment:
Do we need the timezone information? I figured let's just store all timestamps in UTC which should save us a few bytes and any timezone-related headaches.

Reply:
Both types take up the same amount of space in the database, and Postgres does not store the timezone information. It's just aware of the timezone if we happen to pass it a timestamp with a non-UTC offset (and so stores the correct instant in time). Since everything is probably in UTC, this isn't an issue, but I only see benefits by using `TIMESTAMPTZ`, hence the change.